mlflow/data/huggingface_dataset.py

```diff
@@ -191,8 +191,10 @@ def from_huggingface(
 
     :param ds: A Hugging Face dataset. Must be an instance of `datasets.Dataset`.
                Other types, such as `datasets.DatasetDict`, are not supported.
-    :param path: The path of the Hugging Face dataset. This is used by the `datasets.load_dataset()`
-                 function to reload the dataset upon request via `HuggingFaceDataset.source.load()`.
+    :param path: The path of the Hugging Face dataset used to construct the source. This is used by
+                 the `datasets.load_dataset()` function to reload the dataset upon request via
+                 `HuggingFaceDataset.source.load()`. If no path is specified,
+                 a CodeDatasetSource is used.
```

Comment:
Can we explain what `CodeDatasetSource` does in the docstrings? Might be hard for users to interpret otherwise.